android/app/src/main/java/app/organicmaps/util/UiUtils.java

```diff
@@ -142,7 +143,22 @@ public static void visibleIf(boolean condition, View... views)
 
   public static void showIf(boolean condition, View view)
   {
-    view.setVisibility(condition ? View.VISIBLE : View.GONE);
+    if (view == null)
+      return;
+
+    if (condition)
+    {
+      view.setVisibility(View.VISIBLE);
+
+      if (view instanceof LinearProgressIndicator)
+      {
+        ((LinearProgressIndicator) view).setIndeterminate(true);
+      }
```

Comment:
From my POV, it is a generic function to make such specific things like this ..
Make this ```LinearProgressIndicator``` handling in place, e.g. in a Search View?